cargo-nextest/src/dispatch.rs

```diff
@@ -696,6 +696,16 @@ struct TestReporterOpts {
         env = "NEXTEST_FINAL_STATUS_LEVEL"
     )]
     final_status_level: Option<FinalStatusLevelOpt>,
+
+    /// Visibility of the progress bar.
+    #[clap(
+        long,
+        possible_values = &["true", "false"],
+        conflicts_with_all = &["no-capture", "no-run"],
+        value_name = "HIDE",
+        env = "NEXTEST_HIDE_PROGRESS_BAR"
+    )]
+    hide_progress_bar: Option<bool>,
```

Comment:
I think this can just be a boolean flag that defaults to false. No need for `--hide-progress-bar=true` or `=false`.